### PR TITLE
feat: Add www.iab.org K8s configuration

### LINKED ISSUE #479: Add K8s Kustomize configuration for www.iab.org
To replace the existing Helm chart

### PR DESCRIPTION
**Pull request by microamp**
~~WIP~~ Ready for review now

Resolves #479 

---

Using the existing `k8s/ietfweb` as a base.

This will give you a better view for the diffs:
- ~https://github.com/microamp/www/compare/ce80e80..1def680~ Update: https://github.com/microamp/www/compare/ce80e80..61b2de4

### CODE REVIEW

**Review by kesara — APPROVED** [review]

File: k8s/iabweb/memcached.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+                - ALL
+            readOnlyRootFilesystem: true
+            runAsUser: 65534  # nobody
+            runAsGroup: 65534  # nobody
+        - image: "memcached:1.6-alpine"
+          imagePullPolicy: IfNotPresent
+          args: ["-m", "1024"]
```
Comment (NGPixel) [inline k8s/iabweb/memcached.yaml:36]: Does it actually need 1GB? Consider using a smaller value.

**Review by NGPixel — APPROVED** [review]

File: k8s/iabweb/memcached.yaml (hunk 1 above)
Comment (microamp) [reply k8s/iabweb/memcached.yaml]: Reduced to 256 MB.

**Comment by microamp** [comment]
New commits have been pushed including adding a `secretRef` for External Secrets. Re-requesting review.

File: k8s/iabweb/wagtail.yaml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            - name: "DJANGO_SETTINGS_MODULE"
+              value: "ietf.settings.production"
+            # ensures the pod gets recreated on every deploy:
+            - name: "DEPLOY_UID"
+              value: "$DEPLOY_UID"
+          envFrom:
+            - configMapRef:
+                name: iabwww-django-config
+            - secretRef:
+                name: iabwww-secrets-env
```
Comment (kesara) [inline k8s/iabweb/wagtail.yaml:61]: Don't you need a `secrete-env` defined in this dir?

Comment (kesara) [reply k8s/iabweb/wagtail.yaml:61]: Similar to https://github.com/ietf-tools/datatracker/blob/main/k8s/secrets.yaml

Comment (microamp) [reply k8s/iabweb/wagtail.yaml:61]: I'm happy to add `secrets.yaml` with a placeholder for each required variable in the secret.

On a side note, please note that we have both a config map and an external secret here. We may choose to consolidate all variables into the latter in the future, if that's preferred.

Comment (microamp) [reply k8s/iabweb/wagtail.yaml:61]: Updated: https://github.com/ietf-tools/www/pull/477/commits/61b2de443b6d7df39db377a98cfe990d1ff08992

**Review by kesara — APPROVED** [review]

**Pull request merged**